command/windowsOSCommand.go

```diff
@@ -0,0 +1,74 @@
+package command
+
+import (
+	"path"
+
+	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+var _ osCommand = (*windowsOSCommand)(nil)
+
+type windowsOSCommand struct{}
+
+func newWindowsOSCommand() osCommand {
+	return windowsOSCommand{}
+}
+
+func (fs windowsOSCommand) CreateDirectory(
+	runner *Runner,
+	name string,
+	remotePath pulumi.StringInput,
+	_ bool,
+	opts ...pulumi.ResourceOption) (*remote.Command, error) {
+	useSudo := false
+	return createDirectory(
+		runner,
+		name,
+		"New-Item -Path %s -ItemType Directory -Force",
+		"Remove-Item -Path %s -Force -ErrorAction SilentlyContinue -Recurse",
+		remotePath,
+		useSudo,
+		opts...)
+}
+
+func (fs windowsOSCommand) CopyInlineFile(
+	runner *Runner,
+	name string,
+	fileContent pulumi.StringInput,
+	remotePath string,
+	useSudo bool,
+	append bool,
+	opts ...pulumi.ResourceOption) (*remote.Command, error) {
+	flags := ""
+	if append {
+		flags = "-Append"
+	}
+
+	createCmd := pulumi.Sprintf(`[System.Console]::In.ReadToEnd() | Out-File %v -FilePath %v`, flags, remotePath)
```

Comment:
I have to update this line to handle binary file.

Reply:
`ReadToEnd()` is only for text files ?

Reply:
`ReadToEnd()` handles only string.